content/en/tracing/service_catalog/_index.md

```diff
@@ -88,6 +88,8 @@ You can change the default environment in **APM > Setup & Configuration > Settin
 
 {{< img src="tracing/service_catalog/svc-cat-perf-view.png" alt="Performance view filtered on env:* and scoped to cluster-name:*" style="width:100%;" >}}
 
+The perfomrance metrics are tied to services' [primary operations][14]. If a service is only detected by APM, the performance view shows the APM [trace metrics][13]. If a service is only detected by USM, the performance view shows the [USM metrics][15]. If a service is detected by USM and APM, the performance view shows the the Trace Metrics instead of USM metrics.
```

Comment:
```suggestion
The performance metrics are tied to services' [primary operations][14]. If a service is only detected by APM, the performance view shows the APM [trace metrics][13]. If a service is only detected by USM, the performance view shows the [USM metrics][15]. If a service is detected by USM and APM, the performance view shows the the Trace Metrics instead of USM metrics.
```